### PR TITLE
Send onboarding email during installation

### PR DESCRIPTION
**Pull request by tsteur**
Refs MAR-777

image

In my case it went into spam since the sender was basically `foo@mbp.localdomain`. Not sure how often sending the mail actually works when no proper SMTP is configured and how often it would go to spam? Any thoughts @mattab @Findus23 

I'm catching any error that happens so hopefully this doesn't influence

### CODE REVIEW

**Comment by Findus23** [comment]
In theory it should never matter if you are using SMTP or PHP mail (and therefore `sendmail`), at least once we are using PHPMailer for both.

But maybe we should make the fact that the from address can not be detected automatically and this FAQ more prominently. Maybe even add a system check that looks up if the from mail ends with `.localdomain` or something like this.

https://matomo.org/faq/how-to/faq_109/

File: plugins/Installation/Onboarding.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<br>
+';
+
+        $mail = new Mail();
+        $mail->addTo($email);
+        $mail->setSubject('3 quick wins to improve your Matomo right now');
+        $mail->setBodyHtml($message);
+
+        $testMode = (defined('PIWIK_TEST_MODE') && PIWIK_TEST_MODE);
+        if (!$testMode) {
```
Comment (sgiehl) [inline plugins/Installation/Onboarding.php:70]: That should actually not be needed, as it's already done in the mail class. See 
https://github.com/matomo-org/matomo/blob/da92367838fa85b806dfd2486b6e91c308cb5e46/core/Mail.php#L127-L135

File: plugins/Installation/Onboarding.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
```
Comment (sgiehl) [inline plugins/Installation/Onboarding.php:3]:
```suggestion
 * Matomo - free/libre analytics platform
```

**Review by sgiehl — CHANGES_REQUESTED** [review]

**Comment by tsteur** [comment]
Made the changes @sgiehl

**Comment by tsteur** [comment]
@sgiehl I suppose it is good to merge now?

**Comment by Findus23** [comment]
I still think that the subject line looks like a random SPAM mail.

**Comment by tsteur** [comment]
@Findus23 changed it 👍

**Review by sgiehl — APPROVED** [review]
Tested that locally again and it seems to work as expected

**Pull request merged**